src/test/java/edu/harvard/iq/dataverse/api/DatasetsIT.java

```diff
@@ -668,13 +644,22 @@ public void testDatasetVersionsAPI() {
         
         // Now check that the file is NOT shown, when we ask the versions api to 
         // skip files: 
-        boolean skipFiles = true; 
-        unpublishedDraft = UtilIT.getDatasetVersion(datasetPid, DS_VERSION_DRAFT, apiToken, skipFiles, false);
+        boolean excludeFiles = true; 
+        unpublishedDraft = UtilIT.getDatasetVersion(datasetPid, DS_VERSION_DRAFT, apiToken, excludeFiles, false);
         unpublishedDraft.prettyPrint();
         unpublishedDraft.then().assertThat()
                 .body("data.files", equalTo(null))
                 .statusCode(OK.getStatusCode());
 
+        excludeFiles = false; 
+        unpublishedDraft = UtilIT.getDatasetVersion(datasetPid, DS_VERSION_DRAFT, apiToken, excludeFiles, false);
+        unpublishedDraft.prettyPrint();
+        unpublishedDraft.then().assertThat()
+                .body("data.files.size()", equalTo(1))
+                .statusCode(OK.getStatusCode());
```

Comment:
statusCode should be checked before body.

Otherwise you don't get a meaningful error when statusCode is an unexpected value.

Basically, statusCode should always be the first line.

Reply:
I am fixing this, thanks! I am also going to fix it on other places since this was code I copied. 😓